### DISCUSSION TITLE
Typo on the pricing page of codemagic.io

### DISCUSSION THREAD

**Discussion by lig** [codemagic-ci-cd/codemagic-docs#1356] [Q&A] [answered]
https://codemagic.io/pricing/

`upgadable` -> `upgradable`

![Screenshot-20230127165413-340x38](https://user-images.githubusercontent.com/38705/215131585-eb92f6f3-8ccb-44b9-9257-77292bfc9e18.png)

**Comment by helinanever** [comment, marked as answer]
Thank you for pointing this out, this has been fixed now.
